From 3754c48ec570c5c08fa05c46a31b00e961abbf96 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Fri, 13 Dec 2019 11:51:43 -0500
Subject: [PATCH] Fix help message for --client-key-file

Signed-off-by: Stephen Gallagher <sgallagh@redhat.com>
---
 src/sscg.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/sscg.c b/src/sscg.c
index 02e690860e092d4f154c74cd2eaf03ac0392f31b..035e89882e85ad07e4bf60f62cc95509bf2e31a9 100644
--- a/src/sscg.c
+++ b/src/sscg.c
@@ -737,9 +737,8 @@ main (int argc, const char **argv)
       &client_key_file,
       0,
       _ ("Path where the client's private key will be stored. "
-         "(default is client-file with a .key suffix, if "
-         "--client-file was passed, otherwise this file will not "
-         "be generated.)"),
+         "(default is the client-file, if --client-file was passed, "
+         "otherwise this file will not be generated.)"),
       NULL,
     },
 
-- 
2.23.0

